meson.build

```diff
@@ -367,15 +367,19 @@ if not lib_crypt.found()
   lib_crypt = cc.find_library('crypt', required : get_option('build-sulogin'))
 endif
 
+pam_req = get_option('build-login').enabled() or \
```

Comment:
I'd name the var `pam_required` to not confuse it with "pam_requirement".

Reply:
Done.